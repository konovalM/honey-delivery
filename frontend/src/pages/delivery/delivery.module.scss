@use '@styles/colors.scss' as *;

.container {
    max-width: 1200px;
    margin: 0 auto;
    padding: 2rem 0;
}

.mainTitle {
    text-align: center;
    color: $text-base;
    margin-bottom: 1.5rem !important;
}

.introText {
    text-align: center;
    font-size: 1.1rem;
    max-width: 800px;
    margin: 0 auto 2rem !important;
    color: #555;
}

.highlightBanner {
    background: #fff7e6;
    border-left: 4px solid #faad14;
    padding: 1rem;
    margin: 1.5rem 0;
    border-radius: 4px;
}

.highlightText {
    font-size: 1.1rem !important;
    color: #d48806 !important;
    margin: 0 !important;
}

.divider {
    margin: 2rem 0 !important;
    font-size: 1.2rem !important;
    color: #389e0d !important;
}

.deliveryMethods {
    display: grid;
    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
    gap: 2rem;
    margin-bottom: 3rem;
}

.deliveryCard {
    border-radius: 12px !important;
    overflow: hidden;
    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.08);
    transition:
        transform 0.3s ease,
        box-shadow 0.3s ease;

    &:hover {
        transform: translateY(-5px);
        box-shadow: 0 6px 16px rgba(0, 0, 0, 0.12);
    }
}

.imageContainer {
    height: 200px;
    overflow: hidden;
    display: flex;
    align-items: center;
    justify-content: center;
    background: #f7f7f7;
    position: relative;
}

.deliveryImage {
    width: 100%;
    height: 100%;
    object-fit: cover;
}

.cityTag {
    position: absolute;
    top: 10px;
    right: 10px;
    font-weight: bold;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
}

.cardTitle {
    color: #2c3e50 !important;
    margin-bottom: 1rem !important;
}

.sectionTitle {
    color: #389e0d !important;
    margin-bottom: 0.5rem !important;
}

.featuresDivider,
.conditionsDivider,
.noteDivider {
    margin: 1rem 0 !important;
}

.featuresList,
.conditionsList {
    width: 100%;
}

.featureItem,
.conditionItem {
    display: flex;
    align-items: center;
    font-size: 1rem;
}

.noteText {
    font-size: 0.9rem !important;
    display: block;
}

.additionalInfo {
    background: #f6ffed;
    padding: 1.5rem;
    border-radius: 8px;
    margin-top: 2rem;
    border-left: 4px solid #389e0d;
}

.infoTitle {
    color: #389e0d !important;
    margin-bottom: 1rem !important;
}

@media (max-width: 768px) {
    .deliveryMethods {
        grid-template-columns: 1fr;
    }

    .mainTitle {
        font-size: 1.8rem !important;
    }

    .introText,
    .highlightText {
        font-size: 1rem !important;
    }

    .imageContainer {
        height: 160px;
    }
}
