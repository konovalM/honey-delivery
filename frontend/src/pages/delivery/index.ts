export { Delivery as default } from './delivery';
