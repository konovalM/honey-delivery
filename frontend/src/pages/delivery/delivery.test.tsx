import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { Delivery } from './delivery';

describe('Delivery Page', () => {
  it('рендерится без ошибок и соответствует snapshot', () => {
    const { container } = render(<Delivery />);
    expect(container).toMatchSnapshot();
  });
});
