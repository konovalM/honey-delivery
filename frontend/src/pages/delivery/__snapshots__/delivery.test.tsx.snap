// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Delivery Page > —Ä–µ–Ω–¥–µ—Ä–∏—Ç—Å—è –±–µ–∑ –æ—à–∏–±–æ–∫ –∏ —Å–æ–æ—Ç–≤–µ—Ç—Å—Ç–≤—É–µ—Ç snapshot 1`] = `
<div>
  <div
    class="_container_d1b079"
  >
    <h1
      class="ant-typography _mainTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
    >
      –£—Å–ª–æ–≤–∏—è –¥–æ—Å—Ç–∞–≤–∫–∏
    </h1>
    <div
      class="ant-typography _introText_d1b079 css-dev-only-do-not-override-1d4w9r2"
    >
      –ú—ã –ø—Ä–µ–¥–ª–∞–≥–∞–µ–º –Ω–µ—Å–∫–æ–ª—å–∫–æ —É–¥–æ–±–Ω—ã—Ö —Å–ø–æ—Å–æ–±–æ–≤ –¥–æ—Å—Ç–∞–≤–∫–∏ –Ω–∞—à–µ–≥–æ –º–µ–¥–∞. –í—ã–±–µ—Ä–∏—Ç–µ –ø–æ–¥—Ö–æ–¥—è—â–∏–π –≤–∞—Ä–∏–∞–Ω—Ç –ø—Ä–∏ –æ—Ñ–æ—Ä–º–ª–µ–Ω–∏–∏ –∑–∞–∫–∞–∑–∞.
    </div>
    <div
      class="_highlightBanner_d1b079"
    >
      <span
        class="ant-typography _highlightText_d1b079 css-dev-only-do-not-override-1d4w9r2"
      >
        <strong>
          üöö –ë–µ—Å–ø–ª–∞—Ç–Ω–∞—è –¥–æ—Å—Ç–∞–≤–∫–∞ –ø–æ –°–ü–± –∏ –ö—Ä–∞—Å–Ω–æ–¥–∞—Ä—É –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –æ—Ç 3000 ‚ÇΩ | –ü–æ –†–æ—Å—Å–∏–∏ - –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –æ—Ç 10 –∫–≥ –∏–ª–∏ 10 999 ‚ÇΩ
        </strong>
      </span>
    </div>
    <div
      class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-with-text ant-divider-with-text-start _divider_d1b079"
      role="separator"
    >
      <span
        class="ant-divider-inner-text"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
            –î–æ—Å—Ç—É–ø–Ω—ã–µ —Å–ø–æ—Å–æ–±—ã
          </strong>
        </span>
      </span>
    </div>
    <div
      class="_deliveryMethods_d1b079"
    >
      <div
        class="ant-card ant-card-bordered _deliveryCard_d1b079 css-dev-only-do-not-override-1d4w9r2"
      >
        <div
          class="ant-card-cover"
        >
          <div
            class="_imageContainer_d1b079"
          >
            <div
              class="ant-image css-dev-only-do-not-override-1d4w9r2"
            >
              <img
                alt="–ö—É—Ä—å–µ—Ä—Å–∫–∞—è –¥–æ—Å—Ç–∞–≤–∫–∞"
                class="ant-image-img _deliveryImage_d1b079 css-dev-only-do-not-override-1d4w9r2"
                src="https://static.vecteezy.com/system/resources/thumbnails/004/379/352/small_2x/delivery-van-driver-with-parcels-flat-character-courier-postman-deliveryman-holding-cardboard-boxes-and-order-receipt-isolated-cartoon-illustration-on-white-background-shipping-service-transport-vector.jpg"
              />
            </div>
            <span
              class="ant-tag ant-tag-has-color _cityTag_d1b079 css-dev-only-do-not-override-1d4w9r2"
              style="background-color: rgb(24, 144, 255);"
            >
              –¢–æ–ª—å–∫–æ –°–ü–± –∏ –ö—Ä–∞—Å–Ω–æ–¥–∞—Ä
            </span>
          </div>
        </div>
        <div
          class="ant-card-body"
        >
          <h3
            class="ant-typography _cardTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –ö—É—Ä—å–µ—Ä—Å–∫–∞—è –¥–æ—Å—Ç–∞–≤–∫–∞
          </h3>
          <div
            class="ant-typography css-dev-only-do-not-override-1d4w9r2"
          >
            –ë—ã—Å—Ç—Ä–∞—è –¥–æ—Å—Ç–∞–≤–∫–∞ –ø–æ –°–∞–Ω–∫—Ç-–ü–µ—Ç–µ—Ä–±—É—Ä–≥—É –∏ –ö—Ä–∞—Å–Ω–æ–¥–∞—Ä—É
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _featuresDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –û—Å–æ–±–µ–Ω–Ω–æ—Å—Ç–∏:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _featuresList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –î–æ—Å—Ç–∞–≤–∫–∞ –Ω–∞ —Å–ª–µ–¥—É—é—â–∏–π –¥–µ–Ω—å –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –¥–æ 18:00
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –í–æ–∑–º–æ–∂–Ω–æ—Å—Ç—å –≤—ã–±—Ä–∞—Ç—å —É–¥–æ–±–Ω—ã–π –≤—Ä–µ–º–µ–Ω–Ω–æ–π –∏–Ω—Ç–µ—Ä–≤–∞–ª
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –û–ø–ª–∞—Ç–∞ –ø—Ä–∏ –ø–æ–ª—É—á–µ–Ω–∏–∏ (–Ω–∞–ª–∏—á–Ω—ã–º–∏ –∏–ª–∏ –∫–∞—Ä—Ç–æ–π)
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _conditionsDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –£—Å–ª–æ–≤–∏—è:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _conditionsList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                <span
                  class="ant-typography css-dev-only-do-not-override-1d4w9r2"
                >
                  –î–æ—Å—Ç—É–ø–Ω–æ —Ç–æ–ª—å–∫–æ –¥–ª—è 
                  <span
                    class="ant-typography css-dev-only-do-not-override-1d4w9r2"
                  >
                    <strong>
                      –°–ü–± –∏ –ö—Ä–∞—Å–Ω–æ–¥–∞—Ä–∞
                    </strong>
                  </span>
                </span>
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –ë–µ—Å–ø–ª–∞—Ç–Ω–æ –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –æ—Ç 3000 ‚ÇΩ
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –°—Ç–æ–∏–º–æ—Å—Ç—å: 300 ‚ÇΩ (–¥–ª—è –∑–∞–∫–∞–∑–æ–≤ –¥–æ 3000 ‚ÇΩ)
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _noteDivider_d1b079"
            role="separator"
          />
          <span
            class="ant-typography ant-typography-secondary _noteText_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            ‚ÑπÔ∏è 
            –ö—É—Ä—å–µ—Ä –ø–æ–∑–≤–æ–Ω–∏—Ç –∑–∞ 30-60 –º–∏–Ω—É—Ç –¥–æ –¥–æ—Å—Ç–∞–≤–∫–∏
          </span>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered _deliveryCard_d1b079 css-dev-only-do-not-override-1d4w9r2"
      >
        <div
          class="ant-card-cover"
        >
          <div
            class="_imageContainer_d1b079"
          >
            <div
              class="ant-image css-dev-only-do-not-override-1d4w9r2"
            >
              <img
                alt="–°–î–≠–ö"
                class="ant-image-img _deliveryImage_d1b079 css-dev-only-do-not-override-1d4w9r2"
                src="https://www.molnet.ru/mos/image?objectId=825319&trim_x=660"
              />
            </div>
          </div>
        </div>
        <div
          class="ant-card-body"
        >
          <h3
            class="ant-typography _cardTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –°–î–≠–ö
          </h3>
          <div
            class="ant-typography css-dev-only-do-not-override-1d4w9r2"
          >
            –ù–∞–¥–µ–∂–Ω–∞—è –¥–æ—Å—Ç–∞–≤–∫–∞ –ø–æ –≤—Å–µ–π –†–æ—Å—Å–∏–∏
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _featuresDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –û—Å–æ–±–µ–Ω–Ω–æ—Å—Ç–∏:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _featuresList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –ë–æ–ª–µ–µ 5000 –ø—É–Ω–∫—Ç–æ–≤ –≤—ã–¥–∞—á–∏ –ø–æ —Å—Ç—Ä–∞–Ω–µ
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –í–æ–∑–º–æ–∂–Ω–æ—Å—Ç—å –¥–æ—Å—Ç–∞–≤–∫–∏ –¥–æ –¥–≤–µ—Ä–∏
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –¢—Ä–µ–∫–∏–Ω–≥ –∑–∞–∫–∞–∑–∞ –≤ –º–æ–±–∏–ª—å–Ω–æ–º –ø—Ä–∏–ª–æ–∂–µ–Ω–∏–∏
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _conditionsDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –£—Å–ª–æ–≤–∏—è:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _conditionsList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –ë–µ—Å–ø–ª–∞—Ç–Ω–æ –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –æ—Ç 10 –∫–≥ –∏–ª–∏ —Å—Ç–æ–∏–º–æ—Å—Ç–∏ –æ—Ç 10 999 ‚ÇΩ
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –°—Ç–æ–∏–º–æ—Å—Ç—å: –æ—Ç 450 ‚ÇΩ (—Ä–∞—Å—Å—á–∏—Ç—ã–≤–∞–µ—Ç—Å—è –∏–Ω–¥–∏–≤–∏–¥—É–∞–ª—å–Ω–æ)
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –°—Ä–æ–∫–∏: 2-7 –¥–Ω–µ–π –≤ –∑–∞–≤–∏—Å–∏–º–æ—Å—Ç–∏ –æ—Ç —Ä–µ–≥–∏–æ–Ω–∞
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _noteDivider_d1b079"
            role="separator"
          />
          <span
            class="ant-typography ant-typography-secondary _noteText_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            ‚ÑπÔ∏è 
            –†–µ–∫–æ–º–µ–Ω–¥—É–µ–º –≤—ã–±–∏—Ä–∞—Ç—å –¥–ª—è –∫—Ä—É–ø–Ω—ã—Ö –∑–∞–∫–∞–∑–æ–≤
          </span>
        </div>
      </div>
      <div
        class="ant-card ant-card-bordered _deliveryCard_d1b079 css-dev-only-do-not-override-1d4w9r2"
      >
        <div
          class="ant-card-cover"
        >
          <div
            class="_imageContainer_d1b079"
          >
            <div
              class="ant-image css-dev-only-do-not-override-1d4w9r2"
            >
              <img
                alt="–ü–æ—á—Ç–∞ –†–æ—Å—Å–∏–∏"
                class="ant-image-img _deliveryImage_d1b079 css-dev-only-do-not-override-1d4w9r2"
                src="https://vlfin.ru/image/720/0?filename=/upload/iblock/495/4952f9d6366634ccb9d02ddb38cacc4d.jpg"
              />
            </div>
          </div>
        </div>
        <div
          class="ant-card-body"
        >
          <h3
            class="ant-typography _cardTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –ü–æ—á—Ç–∞ –†–æ—Å—Å–∏–∏
          </h3>
          <div
            class="ant-typography css-dev-only-do-not-override-1d4w9r2"
          >
            –î–æ—Å—Ç–∞–≤–∫–∞ –≤ –ª—é–±—É—é —Ç–æ—á–∫—É —Å—Ç—Ä–∞–Ω—ã
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _featuresDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –û—Å–æ–±–µ–Ω–Ω–æ—Å—Ç–∏:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _featuresList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –î–æ—Å—Ç—É–ø–Ω–æ –¥–∞–∂–µ –≤ –æ—Ç–¥–∞–ª–µ–Ω–Ω—ã–µ —Ä–µ–≥–∏–æ–Ω—ã
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –û—Ç–ø—Ä–∞–≤–∫–∞ –¥–µ–Ω—å –≤ –¥–µ–Ω—å –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –¥–æ 14:00
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _featureItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                ‚Ä¢ 
                –ü–æ–ª—É—á–µ–Ω–∏–µ –≤ –±–ª–∏–∂–∞–π—à–µ–º –ø–æ—á—Ç–æ–≤–æ–º –æ—Ç–¥–µ–ª–µ–Ω–∏–∏
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _conditionsDivider_d1b079"
            role="separator"
          />
          <h5
            class="ant-typography _sectionTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            –£—Å–ª–æ–≤–∏—è:
          </h5>
          <div
            class="ant-space css-dev-only-do-not-override-1d4w9r2 ant-space-vertical ant-space-gap-row-small ant-space-gap-col-small _conditionsList_d1b079"
          >
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –ë–µ—Å–ø–ª–∞—Ç–Ω–æ –ø—Ä–∏ –∑–∞–∫–∞–∑–µ –æ—Ç 10 –∫–≥ –∏–ª–∏ —Å—Ç–æ–∏–º–æ—Å—Ç–∏ –æ—Ç 10 999 ‚ÇΩ
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –°—Ç–æ–∏–º–æ—Å—Ç—å: –æ—Ç 350 ‚ÇΩ (–ø–æ —Ç–∞—Ä–∏—Ñ–∞–º –ü–æ—á—Ç—ã –†–æ—Å—Å–∏–∏)
              </span>
            </div>
            <div
              class="ant-space-item"
            >
              <span
                class="ant-typography _conditionItem_d1b079 css-dev-only-do-not-override-1d4w9r2"
              >
                üìå 
                –°—Ä–æ–∫–∏: 5-14 –¥–Ω–µ–π –≤ –∑–∞–≤–∏—Å–∏–º–æ—Å—Ç–∏ –æ—Ç —Ä–µ–≥–∏–æ–Ω–∞
              </span>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-1d4w9r2 ant-divider-horizontal ant-divider-dashed _noteDivider_d1b079"
            role="separator"
          />
          <span
            class="ant-typography ant-typography-secondary _noteText_d1b079 css-dev-only-do-not-override-1d4w9r2"
          >
            ‚ÑπÔ∏è 
            –û—Ç–ø—Ä–∞–≤–ª—è–µ–º —Å –æ–±—ä—è–≤–ª–µ–Ω–Ω–æ–π —Ü–µ–Ω–Ω–æ—Å—Ç—å—é –∏ —Ç—Ä–µ–∫-–Ω–æ–º–µ—Ä–æ–º
          </span>
        </div>
      </div>
    </div>
    <div
      class="_additionalInfo_d1b079"
    >
      <h4
        class="ant-typography _infoTitle_d1b079 css-dev-only-do-not-override-1d4w9r2"
      >
        –î–æ–ø–æ–ª–Ω–∏—Ç–µ–ª—å–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è
      </h4>
      <div
        class="ant-typography css-dev-only-do-not-override-1d4w9r2"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
            –£–ø–∞–∫–æ–≤–∫–∞:
          </strong>
        </span>
         –í—Å–µ –∑–∞–∫–∞–∑—ã —Ç—â–∞—Ç–µ–ª—å–Ω–æ —É–ø–∞–∫–æ–≤—ã–≤–∞—é—Ç—Å—è –≤ —Å–ø–µ—Ü–∏–∞–ª—å–Ω—É—é —Ç–µ—Ä–º–æ—É–ø–∞–∫–æ–≤–∫—É –¥–ª—è —Å–æ—Ö—Ä–∞–Ω–µ–Ω–∏—è –∫–∞—á–µ—Å—Ç–≤–∞ –ø—Ä–æ–¥—É–∫—Ç–∞ –ø—Ä–∏ —Ç—Ä–∞–Ω—Å–ø–æ—Ä—Ç–∏—Ä–æ–≤–∫–µ.
      </div>
      <div
        class="ant-typography css-dev-only-do-not-override-1d4w9r2"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
            –°—Ä–æ–∫–∏ —Ö—Ä–∞–Ω–µ–Ω–∏—è:
          </strong>
        </span>
         –ü–æ—Å–ª–µ –æ—Ç–ø—Ä–∞–≤–∫–∏ –≤—ã –ø–æ–ª—É—á–∏—Ç–µ –ø–æ–¥—Ä–æ–±–Ω—É—é –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏—é –ø–æ —Ö—Ä–∞–Ω–µ–Ω–∏—é –º–µ–¥–∞. –í —Å—Ä–µ–¥–Ω–µ–º —Å—Ä–æ–∫ –≥–æ–¥–Ω–æ—Å—Ç–∏ - 2 –≥–æ–¥–∞.
      </div>
      <div
        class="ant-typography css-dev-only-do-not-override-1d4w9r2"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
            –ö–æ–Ω—Ç–∞–∫—Ç—ã:
          </strong>
        </span>
         –ü–æ –≤—Å–µ–º –≤–æ–ø—Ä–æ—Å–∞–º –¥–æ—Å—Ç–∞–≤–∫–∏ –∑–≤–æ–Ω–∏—Ç–µ
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
             8 (800) 123-45-67
          </strong>
        </span>
         –∏–ª–∏ –ø–∏—à–∏—Ç–µ –Ω–∞
        <span
          class="ant-typography css-dev-only-do-not-override-1d4w9r2"
        >
          <strong>
             delivery@honeyfamily.ru
          </strong>
        </span>
      </div>
    </div>
  </div>
</div>
`;
